connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/EmailRouteBuilder.java

```diff
@@ -390,7 +390,11 @@ protected HttpEndpointBuilderFactory.HttpEndpointBuilder setUpITEndpoint() {
         // Remove the schema from the url to avoid the
         // "ResolveEndpointFailedException", which complaints about specifying
         // the schema twice.
-        final String fullURL = this.emailConnectorConfig.getItUserServiceURL();
+        //
+        // Set the cookie specification to "browser compatibility" to avoid
+        // getting "cookie rejected" warnings for having a strict cookie
+        // policy, specially when we don't really use those cookies for much.
+        final String fullURL = String.format("%s?httpClient.cookieSpec=compatibility", this.emailConnectorConfig.getItUserServiceURL());
```

Comment:
Is there any reason to prefer 
compatibility over ignoreCookies?

https://camel.apache.org/components/4.0.x/http-component.html#_disabling_cookies

I doubt we'll ever use the cookie returned by IT.

Reply:
You are right, yes. I doubted about ignoring all cookies but you are right...